source/Cosmos.IL2CPU/Cosmos.IL2CPU.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
it has to be net5.0 as we are porting all of cosmos to net5.0 and we are dropping support for older .net versions

Reply:
1) I don't have a .NET 5 User Kit to test with, so my comparison while porting is using exactly the same packages as the last UserKit 20200708. Comparing with a moving target is not feasible (I'm looking at github.com/CosmosOS/Cosmos/releases).

2) There's no reason why the compiler itself (Cosmos.IL2CPU) - which is loaded in the compilation context due to the Assembler workings in plugs - would have to be .NET 5, because ".NET 5 and all future versions will continue to support .NET Standard 2.1 and earlier". 

3) In the Readme file, it's saying: "separating out IL2CPU to allow it to operate as a stand alone project". So, theoretically I could use it in .NET 3.1 LTS, for example and create my completely separate stuff. For independent projects, Microsoft says: "My expectation is that widely used libraries will end up multi-targeting for both .NET Standard 2.0 and .NET 5"
(https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/).

By forcing a not needed version increase, you are limiting the usage of the compiler and preventing separate use cases,
like for example, me using it to combine it with the latest .NET 2.0 UserKit and trying to make that run under Mac & Linux.

Reply:
I figured out how to build net 5 devkit and so I revert my change.